code/modules/admin/topic.dm

```diff
@@ -1287,6 +1287,57 @@
 		NP.ckey = M.ckey
 		qdel(M)
 
+	else if(href_list["eraseflavortext"])
+		if(!check_rights(R_ADMIN))
+			return
+
+		var/mob/M = locateUID(href_list["eraseflavortext"])
+
+		if(!M.client)
+			to_chat(usr, "<span class='warning'>[M] doesn't seem to have an active client.</span>")
+			return
+
+		if(M.flavor_text == "" && M.client.prefs.flavor_text == "")
+			to_chat(usr, "<span class='warning'>[M] has no flavor text set.</span>")
+			return
+
+		if(alert(usr, "Erase [key_name(M)]'s flavor text?", "Message", "Yes", "No") != "Yes")
+			return
+
+		log_admin("[key_name(usr)] has erased [key_name(M)]'s flavor text.")
+		message_admins("[key_name_admin(usr)] has erased [key_name_admin(M)]'s flavor text.")
+
+		// Clears the mob's flavor text
+		M.flavor_text = "";
```

Comment:
what's up with the erroneous `;`

Reply:
The result of knowing more languages than one person should in their lifetime